tests/.vscode/settings.json

```diff
@@ -0,0 +1,31 @@
+{
```

Comment:
is this intended to be part of the PR?

Reply:
I added the file intentionally because all the spellcheck errors get reported as problems in VSCode, so I thought I would just add the dictionary additions to the project. Since personally configuration settings (like window positions) do not seem to be stored there, I thought it would be safe to add this file.

Reply:
After discussion, decided to remove this file.